### ISSUE TITLE
`pyproject-fmt` incompatible with `flake8` 5.0.4?

### ISSUE THREAD

**Issue by Saransh-cpp** [tox-dev/pyproject-fmt#31]
It looks like `pyproject-fmt` is incompatible with the latest release of `flake8`? See https://github.com/scikit-hep/vector/pull/236 and the corresponding CI run - https://github.com/scikit-hep/vector/runs/7741824759?check_suite_focus=true

```
Traceback (most recent call last):
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/bin/pyproject-fmt", line 8, in <module>
    sys.exit(run())
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/pyproject_fmt/__main__.py", line 29, in run
    formatted = format_pyproject(config)
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/pyproject_fmt/formatter/__init__.py", line 25, in format_pyproject
    result = parsed.as_string().rstrip("\n")
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/tomlkit/container.py", line 503, in as_string
    s += self._render_table(k, v)
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/tomlkit/container.py", line 566, in _render_table
    cur += self._render_simple_item(
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/tomlkit/container.py", line 631, in _render_simple_item
    f"{decode(item.as_string())}"
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/tomlkit/items.py", line 1217, in as_string
    return f'[{"".join(v.as_string() for v in self._iter_items())}]'
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/tomlkit/items.py", line 1217, in <genexpr>
    return f'[{"".join(v.as_string() for v in self._iter_items())}]'
  File "/home/runner/.cache/pre-commit/repovmci12e7/py_env-python3/lib/python3.10/site-packages/tomlkit/items.py", line 1194, in _iter_items
    yield from v
TypeError: 'Whitespace' object is not iterable
```

**Comment by abravalheri** [comment]
I am seeing this error independently of flake8 (I think), when I run the following:

```bash
cd /tmp
cat <<EOF > pyproject.toml
[build-system]
requires = ["setuptools>=61.2"]
build-backend = "setuptools.build_meta"

[project]
version = "42"
name = "myproj"
EOF

pip-run 'pyproject-fmt==0.3.4' -- -m pyproject_fmt pyproject.toml
```


<details>
<summary>

#### Error Trace

</summary>

```console
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tmp/pip-run-xw1yirpy/pyproject_fmt/__main__.py", line 55, in <module>
    sys.exit(run())
  File "/tmp/pip-run-xw1yirpy/pyproject_fmt/__main__.py", line 29, in run
    formatted = format_pyproject(config)
  File "/tmp/pip-run-xw1yirpy/pyproject_fmt/formatter/__init__.py", line 25, in format_pyproject
    result = parsed.as_string().rstrip("\n")
  File "/tmp/pip-run-xw1yirpy/tomlkit/container.py", line 503, in as_string
    s += self._render_table(k, v)
  File "/tmp/pip-run-xw1yirpy/tomlkit/container.py", line 566, in _render_table
    cur += self._render_simple_item(
  File "/tmp/pip-run-xw1yirpy/tomlkit/container.py", line 628, in _render_simple_item
    f"{item.trivia.indent}"
  File "/tmp/pip-run-xw1yirpy/tomlkit/items.py", line 1217, in as_string
    return f'[{"".join(v.as_string() for v in self._iter_items())}]'
  File "/tmp/pip-run-xw1yirpy/tomlkit/items.py", line 1217, in <genexpr>
    return f'[{"".join(v.as_string() for v in self._iter_items())}]'
  File "/tmp/pip-run-xw1yirpy/tomlkit/items.py", line 1194, in _iter_items
    yield from v
TypeError: 'Whitespace' object is not iterable
```

</details>
<details>
<summary>

#### System info

</summary>

```console

Ubuntu 20.04.4 LTS
Python 3.8.10 (default, Jun 22 2022, 20:18:18) [GCC 9.4.0]
Package       Version
------------- -------
autocommand   2.2.1
packaging     21.3
path          16.3.0
pip           22.2.2
pip-run       8.8.0
pkg_resources 0.0.0
pyparsing     3.0.9
pyproject-fmt 0.3.4
setuptools    63.4.1
tomlkit       0.11.3
wheel         0.37.1
```
</details>

**Comment by gaborbernat** [comment]
Yeah, the CI is broken we'll need to fix this. I think tomlkit changed...

**Comment by gaborbernat** [comment]
Should be now fixed with 0.3.5
